cmd/buildah/images.go

```diff
@@ -233,7 +232,7 @@ func outputImages(ctx context.Context, images []storage.Image, format string, st
 				createdTime = inspectedTime
 			}
 		}
-		createdTime = util.GetLocalTime(createdTime)
+		createdTime = createdTime.Local()
```

Comment:
Won't this change the behavior?
Before we were calculating the duration when timezone is `UTC` and now we are using the local timezone?

Or I didn't understand the code correctly? :grinning:

Reply:
We were calculating an offset from UTC using the local timezone information and offsetting the value of "now" by that amount.  If the format string we were using for display later in `outputImages()` included a timezone specifier ("MST"), it would have been misleading.

Reply:
Thanks for clarifying this.

LGTM